tests/conftest.py

```diff
@@ -43,8 +43,8 @@ def create_service(client):
     with client.application.app_context():
         # Setup code to create the object in the database
         user, created = get_or_create(db.session, User, username="mehmet") # Adjust fields as necessary
-        get_or_create(db.session, Service, name="Github Demo", type="Github", desc="description", user_id=user.id)
-
+        service, created = get_or_create(db.session, Service, name="Github Demo", type="Github", desc="description", user_id=user.id)
```

Comment:
GitHub is a proper noun.

```suggestion
        service, created = get_or_create(db.session, Service, name="GitHub Demo", type="GitHub", desc="description", user_id=user.id)
```